src/widgetastic_patternfly4/navigation.py

```diff
@@ -25,10 +25,10 @@ class BaseNavigation:
         './/*[self::a or self::button][contains(@class, "pf-m-current") or '
         'parent::li[contains(@class, "pf-m-current")]]'
     )
-    ITEMS = "./ul/li/*[self::a or self::button]"
+    ITEMS = "./ul/li/*[self::a or self::button] | /ul/section/ul/li/*[self::a or self::button]"
```

Comment:
Hm, how can this `/ul/section/ul/li/*[self::a or self::button]` work?

Reply:
In the case of the new insights left nav some of the items are nested under group identifiers.

Reply:
After some discussion I'm thinking this should probably be an override made in platform ui, so I'll close this if that sounds good

Reply:
> In the case of the new insights left nav some of the items are nested under group identifiers.
But you specified an absolute locator and I wonder how it could be found in the DOM.

Reply:
Oh, is that due to forgetting the period? Otherwise I'm not sure exactly what you mean. It did work though

Reply:
Yes, it's about missing period :)